### ISSUE TITLE
[Feature Request] Option to mark Lost Marbles, Rune Switches, Poetry Pages, and Obelisks as complete

### ISSUE THREAD

**Issue by bmfrosty** [abramhindle/ttwl-cli-saveedit#4]
Add options to mark all of these as complete.  These should be the things that a character freshly run through the story is most likely missing and they all involve tromping around various areas to complete.  I've left out lucky dice, bottlecaps, and shrine completion since lucky dice should be handled in profile, bottlecaps are trivial once story is completed, and shrines are similarly likely to be done when the character is going through the story.

It may be reasonable to put guard rails to make sure that the required areas have been at least opened and also to check whether there's another flag thrown on completion to add it to the world buffs.

**Comment by abramhindle** [comment]
Those are all reasonable things to wish for. They aren't in the cli save editor yet but once we add them we can enable it on the web.

**Comment by abramhindle** [comment]
We have lucky dice handled in the profile editor.

**Comment by ghost** [comment]
Any chance you add options for Gold/cash maybe even souls?

**Comment by abramhindle** [comment]
These options are already available

```
  --money MONEY         Set money value (default: None)
  --moon-orbs MOON_ORBS
                        Set Moon Orbs value (default: None)
  --souls SOULS         Set Lost Souls value (default: None)
```

**Comment by ghost** [comment]
sorry about that i didnt realize there where two separate sites for editing. my only question left is, is there a way edit how many points you have for the class itself ? i didnt see the option i might be blind but i dont think i saw it unless it was named something else

**Comment by bmfrosty** [comment]
Points are controlled by level and story progression.  I don't have
details, but I think you end up with 44 points at level 40.  You may end up
with an broken character if you skip the story progression.  Quick note is
that you should be able to boost your character level early on and just
breeze through the story.
You can also make a copy of a character that's already progressed through
the story (look at various youtuber's discords and nexusmods, or just your
own characters) and adjust them fairly easily.  I forget if this tool
includes tools for changing starting classes, but I'm pretty sure you can
change your background.  If it's just background, you can grab characters
that have already made it through the story on nexusmods and adjust their
backgrounds to meet your needs.

**Comment by ghost** [comment]
yes your right 44 points total for any character or class what im trying say is if i wanted for example max out the skill points for BRR-ZERKER AND STABBOMANCER or (any secondary class its not able to because of only having 44 points) back in Borderlands 1 there was a save editor that allowed you to edit the skill points so i wasnt sure if it was possible for this editor im using to do the same. sorry if i didnt explain right the first time

**Comment by ghost** [comment]
any chance this editor you guys made works with borderlands 1 2 and 3 and pre sequel ? im having hard time finding a mod or editor for them that enable more keys?

**Comment by abramhindle** [comment]
Not going to do anything about this.
